pinot-core/src/main/java/org/apache/pinot/core/operator/filter/predicate/PredicateEvaluatorProvider.java

```diff
@@ -77,8 +77,8 @@ public static PredicateEvaluator getPredicateEvaluator(Predicate predicate, Dict
             throw new UnsupportedOperationException("Unsupported predicate type: " + predicate.getType());
         }
       }
-    } catch (NumberFormatException e) {
-      // This NumberFormatException is caused by passing in a non-numeric string as numeric number in query
+    } catch (Exception e) {
```

Comment:
Do we throw exceptions other than `NumberFormatException`?

Reply:
Yes, `BytesUtils.toBytes()` can throw `IllegalArgumentException`